src/views/Download.vue

```diff
@@ -1,77 +1,97 @@
+              Sponge
+            </span>
+            <small>SpongeAPI versions 5-8</small>
+          </a>
+          <a :href="downloads.nukkit" class="resource">
+            <span>
+              <font-awesome icon="arrow-alt-circle-down" />
+              Nukkit
+            </span>
+            <small>Version NukkitX b93 and above</small>
+          </a>
+          <a :href="downloads.velocity" class="resource">
+            <span>
+              <font-awesome icon="arrow-alt-circle-down" />
+              Velocity
+            </span>
+            <small>Version 1.0</small>
+          </a>
+          <a :href="downloads['bukkit-legacy']" class="resource">
+            <span>
+              <font-awesome icon="arrow-alt-circle-down" />
+              Bukkit Legacy
+            </span>
+            <small>Version 1.7.10</small>
+          </a>
+        </div>
 
-      <div>
-        <h2>How to install:</h2>
-        <ol>
-          <li>Place the downloaded <code>.jar</code> file into your server's <code>plugins</code> or <code>mods</code> folder</li>
-          <li>Start or restart your server - <strong>do not reload!</strong></li>
-          <li>Check your <code>plugins</code> or <code>mods</code> folder for a <code>LuckPerms</code> folder - see the tips below if the folder doesn't generate</li>
-          <li>Open the <code>config</code> file in your favourite text editor and adjust any settings to your liking</li>
-        </ol>
-        <h2>Having trouble installing?</h2>
-        <ul>
-          <li>Make sure to check your console for any errors - especially during start up</li>
-          <li>Check the <a href="https://github.com/lucko/LuckPerms/wiki" target="_blank">wiki</a> to see if you missed any important steps during setup</li>
-          <li>Delete the <code>libs</code> folder and restart the server to let it regenerate, sometimes this may fix the problem</li>
-          <li>If all else fails, join our <a href="https://discord.gg/luckperms" target="_blank">Discord</a> to get some support</li>
-        </ul>
+        <div>
+          <h2>How to install:</h2>
+          <ol>
+            <li>Place the downloaded <code>.jar</code> file into your server's <code>plugins</code> or <code>mods</code> folder</li>
+            <li>Start or restart your server - <strong>do not reload!</strong></li>
+            <li>Check your <code>plugins</code> or <code>mods</code> folder for a <code>LuckPerms</code> folder - see the tips below if the folder doesn't generate</li>
+            <li>Open the <code>config</code> file in your favourite text editor and adjust any settings to your liking</li>
+          </ol>
+          <h2>Having trouble installing?</h2>
+          <ul>
+            <li>Make sure to check your console for any errors - especially during start up</li>
+            <li>Check the <a href="https://github.com/lucko/LuckPerms/wiki" target="_blank">wiki</a> to see if you missed any important steps during setup</li>
```

Comment:
Link to `/wiki`?

Reply:
`<router-link to="/wiki"> ... </router-link>`